### PR TITLE
CB-16251 - Enable GA of DL-DH upgrades

### PR DESCRIPTION
**Pull request by pdarvasi**
This commit contains the following:
 - 'GA' option in the upgrade-supported blueprints, no entitlements needed for upgrade, pls check https://docs.cloudera.com/cdp-public-cloud-preview-features/latest/upgrading-data-hubs/upgrading-data-hubs.pdf for more details
 - upgrade min. upgradeable runtime versions per services according to the table below
 - Validation to allow DL upgrade only if _all_ the attached DHs are upgradeable

image

### CODE REVIEW

**Review by keyki — APPROVED** [review]

**Review by tiborpopovics — APPROVED** [review]

**Comment by lnardai** [comment]
I assume we should wait for the review from wolf and bobcat, adding do not merge, feel free to remove if it's ready to merge.

**Comment by keyki** [comment]
> 

We shouldn't wait for bobcat or wolf. I guess they were added since we modified the templates, but it's not something they should review. We have a fairly large test matrix that we've tested. 

QA has 1 additional validation request that will be done in a separate PR today.

**Pull request merged**
